css/index.css

```diff
@@ -47,6 +51,66 @@ table {
 	border-spacing: 0;
 }
 
+.box1{
+    background-color: teal;
+    width: 100px;
+    height: 100px;
+    margin: 20px;
+}
+.box2{
+    background-color: gold;
+    width: 100px;
+    height: 100px;
+    margin: 20px;
```

Comment:
Your CSS classes are nicely indented and very readable, great job with your code organization.